saleor/order/events.py

```diff
@@ -238,6 +238,17 @@ def order_note_added_event(*, order: Order, user: UserType, message: str) -> Ord
     )
 
 
+def order_customer_note_edited_event(
+    *, order: Order, user: UserType, content: str
+) -> OrderEvent:
+    return OrderEvent.objects.create(
+        order=order,
+        type=OrderEvents.CUSTOMER_NOTE_UPDATED,
+        parameters={"content": content},
```

Comment:
```suggestion
        parameters={"message": content},
```

My fault for suggesting to use `content` for the parameters. We only have a resolver for `message` and it would be best to keep it that way.

The parameters for the function are fine like this, just need to change the JSON at this line.